typing/ctype.ml

```diff
@@ -1464,7 +1521,7 @@ let rec copy_sep cleanup_scope fixed free bound visited ty =
   let univars = free ty in
   if TypeSet.is_empty univars then
     if ty.level <> generic_level then ty else
-    let t = newvar () in
+    let t = newvar Type_layout.any in
```

Comment:
`copy_sep` is only called by `instance_poly'`, which will do the overwriting immediately after the call.  So it would have to be `copy_sep` itself consulting the layout, on some other branch of its descent down the type, and I don't see anything in here that does that.  But perhaps I'm missing something?

Reply:
Yes, you're right. But this took me ~5 minutes to reason through. Perhaps add a comment saying basically what you just wrote? Even simpler, it could just be `(* layout not consulted during copy_sep, so Any is safe *)`.